manifest/autoversion/github.go

```diff
@@ -4,25 +4,43 @@ import (
 	"regexp"
 
 	"github.com/cashapp/hermit/errors"
+	"github.com/cashapp/hermit/github"
 	hmanifest "github.com/cashapp/hermit/manifest"
 )
 
 func gitHub(client GitHubClient, autoVersion *hmanifest.AutoVersionBlock) (string, error) {
-	release, err := client.LatestRelease(autoVersion.GitHubRelease)
-	if err != nil {
-		return "", errors.WithStack(err)
+	var (
+		releases []*github.Release
+		err      error
+	)
+	const history = 100
```

Comment:
I think maybe we should make this quite a bit lower, as auto-versioning gets run for every package. It's not unlikely we'll hit API limits. WDYT?

Reply:
API limits are applied on the number of requests isn't it? The [page size](https://github.com/cashapp/hermit/blob/aa6014177df4032f8b5b6b6e2559be2c4e4a60b3/github/api.go#L102) is also 100 so it should be ok?

Reply:
LGTM!